variables.tf

```diff
@@ -79,3 +85,44 @@ variable "server_name" {
   description = "The name of the server"
   default     = "default"
 }
+
+variable "storageClass" {
+  type        = string
+  description = "The name of the server"
+}
+
+variable "database_name" {
+  type        = string
+  description = "The name of the database to be created"
+  default     = "OMS_DB"
```

Comment:
Remove the default value

Reply:
Removed and added in test/stages/variables.tf